apps/coordinator/src/actions/transactionActions.js

```diff
@@ -450,7 +450,7 @@ export function setFeeFromState(outputsTotalSats) {
   };
 }
 
-export function importPSBT(psbtText) {
+export function importPSBT(psbtText, inputs, isRBFedPSBT) {
```

Comment:
I know we've had conversations about this, but I thought `isRBFedPSBT` isn't necessary anymore? The RBF ones should't have signatures anyway and if you manually import you have no way to know if it's RBF'd or not anyway. might as well just skip this for simplicity. 

Also going to put a pin in the inputs argument. It might be necessary given how these need to created, but it would be nice to have a `dispatch(setInputsFromPsbt(psbt))` just as we have the same for outputs and signatures.